WebApplication/Backend/DTO/DateTimeDTO.cs

```diff
@@ -26,5 +26,21 @@ public bool IsPreferredTimeValid(string date)
         {
             return DateTime.Parse(date) > DateTime.Today.AddDays(2);
         }
+
```

Comment:
Please rename this class to DateFromStringConverter or something because DTO in the name is very misleading - Data Transfer Objects are used to just carry information, not implement some logic